.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,125 @@
+name: Bug Report
+
+about: Create a bug report 
+
+title: "[Package Name]: [Brief Description of the Issue]"
+
+labels:
+  - bug
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        #### Please fill the fields to help us solving your bug! Don't forget to add the right label.
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+
+  - type: dropdown
+    id: package
+    attributes:
+      label: which package/s did you face the problem with?
+      multiple: true
+      options:
+        - signer
+        - tfchain_client
+    validations:
+      required: true
+
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Describe your bug!
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. In this environment...
+        1. With this config...
+        1. Run '...'
+        1. See error...
+    validations:
+      required: false
+      
+  - type: dropdown
```

Comment:
- I didn't ask for the networks and farm details and all this in the issue !
- Please remove it